command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Thu, Jul 28, 2016 at 01:02:28PM -0700, Mike Brown wrote:
> From my reading, the runtime-spec does not currently require a client api.

My intention was to require one.  Can you suggest how I should
strengthen this line to make that the case?

> I don't think the purpose of this document is to say all runtime
> implementors must be command line clients and/or must include a
> client of this form for compliance testing.

I think it is ;).  How would you perform compliance testing without
either a command supporting this API or a runtime-spec entry for an
alternative (e.g. HTTP) API?